linked_list/interface.py

```diff
@@ -29,7 +29,7 @@ def __eq__(self, other):
         raise NotImplementedError()
 
     def __ne__(self, other):
-        raise NotImplementedError()
+        return not self == other
```

Comment:
...probably done in a moment of opening the wrong file, oops. Good catch, thanks. We'll fix it. @seg2151